test/unit/plugins.test.js

```diff
@@ -8,33 +8,24 @@
  *       Lewis Daly - lewisd@crosslaketech.com                            *
  **************************************************************************/
 'use strict'
-
-const Sinon = require('sinon')
-
 const { registerPlugins } = require('../../src/plugins')
+const OpenapiBackend = require('@mojaloop/central-services-shared').Util.OpenapiBackend
+const Path = require('path')
+const Handlers = require('../../src/handlers')
 
-let sandbox
 describe('plugins', () => {
-  beforeAll(() => {
-    sandbox = Sinon.createSandbox()
-  })
-
-  afterEach(() => {
-    sandbox.restore()
+  beforeEach(() => {
+    jest.clearAllMocks()
```

Comment:
maybe let setup in `jest.config` automatic clearing of mocks... https://jestjs.io/docs/en/configuration#clearmocks-boolean

Reply:
done